astroid/bases.py

```diff
@@ -273,27 +275,26 @@ def igetattr(
         try:
             context.lookupname = name
             # XXX frame should be self._proxied, or not ?
-            get_attr = self.getattr(name, context, lookupclass=False)
-            yield from _infer_stmts(
-                self._wrap_attr(get_attr, context), context, frame=self
-            )
+            attrs = self.getattr(name, context, lookupclass=False)
```

Comment:
Could you split out these changes into a separate commit/PR? I'd like to see the effect of these changes on the tests by itself.